### ISSUE TITLE
"EU_events": syntax error in "INSERT INTO mixpanel_Tiimo EU_events

### ISSUE THREAD

**Issue by JMCPH** [mixpanel/mixpanel-swift#556]
We are experiencing an issue on version `3.5.1`.

The tracking events are not showing up in Mixpanel Dashboard, and we are getting an syntax error in the debug log from MixPanel. To resolve the issue we have downgraded to version `3.3.0`.

**Log**
```
2022-08-30 11:19:25.364498+0200 tiimo[32188:2134794] [logging] near "EU_events": syntax error in "INSERT INTO mixpanel_Tiimo EU_events (data, flag, time) VALUES(?, ?, ?);"
2022-08-30 11:19:25.367819+0200 tiimo[32188:2134794] [logging] near "EU_events": syntax error in "CREATE TABLE IF NOT EXISTS mixpanel_Tiimo EU_events(id integer primary key autoincrement,data blob,time real,flag integer);"
2022-08-30 11:19:25.367996+0200 tiimo[32188:2134794] [logging] near "EU_people": syntax error in "CREATE TABLE IF NOT EXISTS mixpanel_Tiimo EU_people(id integer primary key autoincrement,data blob,time real,flag integer);"
2022-08-30 11:19:25.368181+0200 tiimo[32188:2134794] [logging] near "EU_groups": syntax error in "CREATE TABLE IF NOT EXISTS mixpanel_Tiimo EU_groups(id integer primary key autoincrement,data blob,time real,flag integer);"
```

**Comment by jaredmixpanel** [comment]
@JMCPH you're getting this error because of the whitespace in your `instanceName` "Tiimo EU", if you remove or replace the space in that string, for example "Tiimo_EU" or "TiimoEU" this error should go away. However, in the next update I'll make sure we handle this so that whitespace in `instanceName`'s doesn't cause this problem. Thanks for bringing it to our attention. I'll leave this issue open until that update is released.

**Comment by jaredmixpanel** [comment]
@JMCPH this issue is fixed in `v4.0.3`
